src/taipy/core/config/data_node_config.py

```diff
@@ -383,6 +397,47 @@ def _configure_json(
         Config._register(section)
         return Config.sections[DataNodeConfig.name][id]
 
+    @staticmethod
+    def _configure_parquet(
+        id: str,
+        default_path: str = None,
+        exposed_type=_DEFAULT_EXPOSED_TYPE,
+        columns: Optional[List[str]] = None,
+        scope=_DEFAULT_SCOPE,
+        cacheable: bool = False,
+        **properties,
+    ):
+        """Configure a new Parquet data node configuration.
+
+        Parameters:
+            id (str): The unique identifier of the new Parquet data node configuration.
+            default_path (str): The default path of the Parquet file.
+            exposed_type: The exposed type of the data read from Parquet file. The default value is `pandas`.
+            columns (Optional[List[str]]): If not None, only these columns will be read from the file.
```

Comment:
I'm just wondering, if this is only applied when reading, is it better to have this `columns` properties on `read()` method?
Since the user might want to read multiple times, each time with different set of columns, and we don't want the user to directly update the Config.

Let me know what you think.

Reply:
Ah yes that didn't occur to me! That's so much better. I guess I will need to override the `DataNode.read` and `DataNode.read_or_raise` methods in `ParquetDataNode` I think

Reply:
Probably. But overriding the abstract class with new property is not the best idea. We should keep it consistent between all DataNode, or keep it, or change it all.

Anyway, in the case of `pandas` expose type, the user can just select the columns from the DataFrame.
With `numpy`, it will be indexing.
And with custom class, it will be defined by the properties of that class.

@jrobinAV @toan-quach Let me know what you think about this.

Reply:
The read method signature cannot change. It is used by taipy scheduler when executing a task. 
So the read method cannot use runtime parameters. Config properties can be used though.

If the user wants to read data providing specific parameters manually, we need to expose another method. It could be a method specific to ParquetDatanode, or we could override the filter method, or both.